index.js

```diff
@@ -0,0 +1,111 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const models = require('./models')
+const app = express()
+
+app.get('/movies', async (request, response) => {
+  const movies = await models.movies.findAll({
+    include: [{
+      all: true
+    }]
+  })
+  response.send(movies)
+})
+
+app.get('/movies/:identifier', async (request, response) => {
+  const movies = await models.movies.findAll({
+    include: [{
+      all: true
+    }],
+    where: {
+      id: request.params.identifier
+    }
+  })
+  response.send(movies)
+})
+
+app.get('/directors/:identifier', async (request, response) => {
+  const director = await models.directors.findAll({
+    include: {
+      model: models.movies
+    },
+    where: {
+      id: request.params.identifier
+    }
+  })
+  response.send(director)
+})
+
+app.get('/genres/:identifier', async (request, response) => {
```

Comment:
According to the README the route is supposed to be `genre` not `genres`, gotta pay attention to the spec.